ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-commands.sh

```diff
@@ -14,3 +14,8 @@ SAFE_CLUSTER_NAME=${UNSAFE_CLUSTER_NAME#"ci-op-"}
 echo "${SAFE_CLUSTER_NAME}" > ${SHARED_DIR}/CLUSTER_NAME
 
 cp "/var/run/cluster-secrets/${CLUSTER_TYPE}/clouds.yaml"  "${SHARED_DIR}/clouds.yaml"
+
+if [ -f "/var/run/cluster-secrets/${CLUSTER_TYPE}/osp-ca.crt" ]; then
+  cp "/var/run/cluster-secrets/${CLUSTER_TYPE}/osp-ca.crt"  "${SHARED_DIR}/osp-ca.crt"
+  sed -i "s+cacert: .*+cacert: ${SHARED_DIR}/osp-ca.crt+" "${SHARED_DIR}/clouds.yaml"
```

Comment:
`"$SHARED_DIR"` is not guaranteed to be constant across pods. This is probably good for testing, but we'll have to change the common ipi-install (and deprovision) steps before merge